dev-lua/luv/luv-1.30.1_p0.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_MIN_VERSION="2.8"
+inherit cmake-utils unpacker
+
+MY_PV="${PV/_p/-}"
+MY_P="${PN}-${MY_PV}"
+
+DESCRIPTION="Bare libuv bindings for lua"
+HOMEPAGE="https://github.com/luvit/luv"
+# XXX: Remember to check this hash between bumps!
+# https://github.com/luvit/luv/tree/master/deps
+LUA_COMPAT_HASH="daebe77a2f498817713df37f0bb316db1d82222f"
+SRC_URI="
+	https://github.com/luvit/${PN}/archive/${MY_PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/keplerproject/lua-compat-5.3/archive/${LUA_COMPAT_HASH}.zip -> ${PN}-lua-compat-${PV}.zip
```

Comment:
Is this due to the lua-5.3 mask in gentoo? I wonder how to proceed if/when the mask gets lifted... maybe depend on slotted lua for now.

Reply:
As I understand it, yes. It's necessary to make the package to compile under Lua 5.1. Should I slot lua to `:0`? I tried slotting to `:5.1` but it all exploded since they're all masked (?!)

Should I slot `luajit` to `:2` as well?

Reply:
Yeah, `:0` or `:*`. With the asterisk `*` I was just afraid this ebuild will explode _once_ they get unmasked, whenever that may be. :0 is the best solution for now.

Reply:
Done :)